src/App.tsx

```diff
@@ -23,16 +28,15 @@ function Input() {
     />
 }
 
-function App() {
-    return <div className="App">
-        <header className="App-header">
-            <img src="./public/logo.svg" className="App-logo" alt="logo" />
-            <p>
-                Edit <code>src/App.js</code> and save to reload.
-            </p>
-            <Input />
-        </header>
-    </div>;
-}
+const App: React.FC = () => (
+	<HashRouter>
+        <Navbar />
+		<Routes>
+			<Route path="/" element={<Home />} />
+			<Route path="/Create" element={<Create />} />
+            <Route path="/Reports" element={<Reports />} />
+		</Routes>
+	</HashRouter>
+);
```

Comment:
you have mixed tabs and whitespace for your indentation. if your editor supports showing whitespace I would suggest that so you can see where your tabs and spaces are.

I am of the mind to use 4 spaces vs tab but that is a general preference.